From: Makoto Kato <m_kato@ga2.so-net.ne.jp>
Date: Sun, 28 Feb 2016 19:26:18 +0900
Subject: Bug 1250403 - Part 1. Define ARCH_CPU_ARM64 instead of
 ARCH_CPU_AARCH64. r=billm

Chromium defines ARCH_CPU_ARM64 and ARCH_CPU_ARM_FAMILY for aarch64.  So we should use it instead.  Because webrtc and sandbox already define it

MozReview-Commit-ID: C7GYnpRryhA
---
 ipc/chromium/src/build/build_config.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/ipc/chromium/src/build/build_config.h b/ipc/chromium/src/build/build_config.h
index 7a4938e..dcec00d 100644
--- a/ipc/chromium/src/build/build_config.h
+++ b/ipc/chromium/src/build/build_config.h
@@ -109,7 +109,8 @@
 #define ARCH_CPU_ALPHA 1
 #define ARCH_CPU_64_BITS 1
 #elif defined(__aarch64__)
-#define ARCH_CPU_AARCH64 1
+#define ARCH_CPU_ARM_FAMILY 1
+#define ARCH_CPU_ARM64 1
 #define ARCH_CPU_64_BITS 1
 #else
 #error Please add support for your architecture in build/build_config.h
